api/src/opentrons/protocol_engine/types/liquid_level_detection.py

```diff
@@ -17,6 +18,14 @@ def serialize_model(self) -> str:
         """Serialize instances of this class as a string."""
         return "SimulatedProbeResult"
 
+    @model_validator(mode="before")
+    @classmethod
+    def model_validator(cls, data: Any) -> Any:
```

Comment:
Nit: Can we use `object` instead of `Any` here? It doesn't matter so much for the return type, but it's meaningfully safer for the `data` arg.


The examples in the Pydantic docs recommend `Any` but I think that's a bad recommendation.

```suggestion
    def validate_model(cls, data: object) -> object:
```

(Also renaming to `validate_model` for symmetry with the existing `serialize_model`)